scripts/test-abnf-cts.rb

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env ruby
+
+require 'abnftt'
+
+parser = ABNF.from_abnf(File.read("draft-ietf-jsonpath-base.abnf"))
+
+require 'json'
+
+cts = JSON.parse(File.read("../jsonpath-compliance-test-suite/cts.json"))
```

Comment:
Hmmm. This won't be found on most people's machines. Do we need a README? I know this script is really only for limited usage, but...

Reply:
Yes, we may need a README.  Alternatively, the test script could fetch right from Github.

Reply:
Fixed in 2d24063